tests/integration/test_github/test_configuration/test_invalid.py

```
@@ -0,0 +1,138 @@
+                 truncate_monitored_files):
+    '''
+    description: Check if the 'github' module detects invalid configurations. For this purpose, the test
+                 will configure that module using invalid configuration settings with different attributes.
+                 Finally, it will verify that error events are generated indicating the source of the errors.
+
+    wazuh_min_version: 4.3.0
+
+    tier: 0
+
+    parameters:
+        - test_configuration:
+            type: data
+            brief: Configuration used in the test.
+        - test_metadata:
+            type: data
+            brief: Configuration cases.
+        - daemons_handler:
+            type: fixture
+            brief: Manages daemons to reset Wazuh.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - configure_local_internal_options:
+            type: fixture
+            brief: Set internal configuration for testing.
+        - truncate_monitored_files:
+            type: fixture
+            brief: Reset the 'ossec.log' file and start a new monitor.
+
+    assertions:
+        - Verify that the 'github' module generates error events when invalid configurations are used.
+
+    input_description: A configuration template (github_integration) is contained in an external YAML file
+                       (wazuh_conf.yaml). That template is combined with different test cases defined in
+                       the module. Those include configuration settings for the 'github' module.
+
+    expected_output:
+        - r'wm_github_read(): ERROR.* Invalid content for tag .*'
+        - r'wm_github_read(): ERROR.* Empty content for tag .*'
+
+    tags:
+        - invalid_settings
+    '''
+    wazuh_log_monitor = FileMonitor(WAZUH_LOG_PATH)
+    
+    # Configuration error -> ValueError raised
+    try:
```

Don't do this. Use `daemons_handler` with ignore errors. Check: https://github.com/wazuh/wazuh/blob/4157_tests_migration/tests/integration/test_analysisd/test_limit_eps/test_configuration.py

---

Fixed with this [commit](https://github.com/wazuh/wazuh/pull/17621/commits/af6aed8a80dc7f1a8b82803e19050fef15d48e7f)